src/swap/v1_1/index.ts

```diff
@@ -383,6 +399,13 @@ function getFixedOutputSwapQuote({
     (assetInAmountMinusFee * FEE_DENOMINATOR) / (FEE_DENOMINATOR - FEE_NUMERATOR);
   const swapFee = assetInAmount - assetInAmountMinusFee;
 
+  if (assetInAmount > inputSupply) {
```

Comment:
Not sure I understand why this is a problem. The user is supplying the pool with additional input assets. I think we only need to check against the output asset, as that is the one gets send from the pool to the user and if the pool don't have enough, it would lead to an error.

Reply:
Yes looks like I misunderstood the case, will fix it 👍